extensions/s3/src/test/java/com/hazelcast/jet/s3/S3TestBase.java

```diff
@@ -168,6 +170,23 @@ void deleteBucket(S3Client client, String bucket) {
         }
     }
 
+    Stream<String> S3ObjectToLines(S3Object o, S3Client client, String bucketName) {
+        try {
+            ResponseInputStream<GetObjectResponse> is = client.getObject(req -> req.bucket(bucketName).key(o.key()), toInputStream());
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(is, CHARSET))) {
+                // materialize the stream, since we can't read it afterwards
+                return reader.lines().collect(Collectors.toList()).stream();
+            } catch (IOException e) {
+                throw new AssertionError("Error reading file ", e);
+            }
+        } catch (NoSuchBucketException e) {
+            logger.warning("S3 side is having eventual consistency issue that it could not" +
+                    " find the key that is listed before. We ignore this issue.");
+            e.printStackTrace();
```

Comment:
you can pass the exception to the `logger.warning` as an argument instead of print stacktrace